cfn/paws-collector.template

```diff
@@ -331,6 +390,8 @@
                                 "        Plaintext: event.ResourceProperties.Plaintext\n",
                                 "    };\n",
                                 "    const kms = new AWS.KMS();\n",
+                                "    const s3 = new AWS.S3();\n",
+                                "    // We will put the stuff in the bucket here tomorrow.\n",
```

Comment:
We will need some conditions. We don't want to put everything into the bucket, only `{"Fn::Equals" : [{"Ref" : "PawsAuthType"}, "s3object"]},`

Reply:
As we discussed we don't need to encrypt it when we put it into the bucket. S3 service will do this for you. 
We only need to encrypt the file which we going to store in /TMP

Reply:
yep absolutely. I'm going to check if it actually needs to put this stuff in there first.